blueprints/one-oe/runtime/one-stack/oci_open_lz_one-oe_iam.auto.tfvars.json

```diff
@@ -61,46 +61,46 @@
                         }
                     },
                     "CMP-UAT-KEY": {
-                        "name": "cmp-uat",
-                        "description": "one-oe-lz Prod Environment Compartment",
+                        "name": "cmp-lzp-uat",
+                        "description": "One-OE UAT Environment Compartment",
                         "children": {
                             "CMP-U-NETWORK-KEY": {
-                                "name": "cmp-u-network",
-                                "description": "one-oe-lz LZ Prod, UAT Environment, Common Network Compartment"                      
+                                "name": "cmp-lzp-uat-network",
+                                "description": "One-OE UAT Workload Environment, Common Network Compartment"                      
                             },
                             "CMP-U-SECURITY-KEY": {
-                                "name": "cmp-u-security",
-                                "description": "one-oe-lz LZ Prod, UAT Environment, Common Security Compartment"                   
+                                "name": "cmp-lzp-uat-security",
+                                "description": "One-OE UAT Workload Environment, Common Security Compartment"                   
                             },
                             "CMP-U-PLATFORM-KEY": {
-                                "name": "cmp-u-platform",
-                                "description": "one-oe-lz LZ Prod, UAT Environment, Common Platform Compartment"                      
+                                "name": "cmp-lzp-uat-platform",
+                                "description": "One-OE UAT Workload Environment, Common Platform Compartment"                      
                             },
                             "CMP-U-PROJECTS-KEY": {
                                 "name": "cmp-u-projects",
-                                "description": "one-oe-lz LZ Prod, UAT Environment, Common Projects Compartment"                     
+                                "description": "One-OE UAT Workload Environment, Common Projects Compartment"                     
                             }
                         }
                     },
                     "CMP-DEV-KEY": {
-                        "name": "cmp-dev",
-                        "description": "one-oe-lz Prod Environment Compartment",
+                        "name": "cmp-lzp-dev",
+                        "description": "One-OE Dev Environment Compartment",
                         "children": {
                             "CMP-D-NETWORK-KEY": {
-                                "name": "cmp-u-network",
-                                "description": "one-oe-lz LZ Prod, DEV Environment, Common Network Compartment"                      
+                                "name": "cmp-lzp-dev-network",
```

Comment:
this was an error and should have been `d` or make consistent with production which currently uses `p`